apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx

```diff
@@ -291,142 +294,162 @@ const NftViewer = ({ route }: Props) => {
     [nftMetadata, nftStatus],
   );
 
+  const [isOpen, setOpen] = useState<boolean>(false);
+  const onCloseModal = useCallback(() => {
```

Comment:
We need to be careful with the usage of `useMemo` `useCallback` `useEffect`. it's actually not a good practice to overuse them and can worsen perfomance
It would be as efficient to not call `useCallback` here

```ts
const onCloseModal = () => setOpen(false);
```

https://kentcdodds.com/blog/usememo-and-usecallback

Reply:
Ok nice, i'll fix it, I was just copy pasting other code.

Reply:
Not true, please keep using useMemo & useCallbacks everywhere.

Reply:
Also it has already been discussed, we're following this ideology: https://attardi.org/why-we-memo-all-the-things/

Reply:
@lvndry confirmed, using these hooks is encouraged in our codebase, nothing blocking here. Let's keep this debate outside of this PR. :wink: